### PR TITLE
GEOMESA-1138 Support for tools auto export/import lists and maps

### PR DESCRIPTION
**Pull request by elahrvivaz**
* Removing 'index=full:index-value=true' from default geometry in sft string (not used)

Signed-off-by: Emilio Lahr-Vivaz <elahrvivaz@ccri.com>

### CODE REVIEW

File: geomesa-tools/examples/ingest/csv/example.csv [hunk 1]
```diff
@@ -1,4 +1,4 @@
 ID,Name,Age,LastSeen,Friends,Lat,Lon
-23623,Harry,20,2015-05-06,"Will, Mark, Suzan",-100.236523,23
-26236,Hermione,25,2015-06-07,"Edward, Bill, Harry",40.232,-53.2356
-3233,Severus,30,2015-10-23,"Tom, Riddle, Voldemort",3,-62.23
\ No newline at end of file
+23623,Harry,20,2015-05-06,"Will, Mark, Suzan","patronus->10,expelliarmus->9",-100.236523,23
+26236,Hermione,25,2015-06-07,"Edward, Bill, Harry","accio->10",40.232,-53.2356
```
Comment (jahhulbert-ccri) [inline geomesa-tools/examples/ingest/csv/example.csv]: +1 for the ref, +2 for correctness...I'm sure that @jdk2pq and @cne1x would understand the importance of correctness

**Comment by elahrvivaz** [comment]
oops, need to fix license header...

**Comment by jahhulbert-ccri** [comment]
f5cdb381cd3c789590aade798b192f44ae746968

**Pull request closed without merging**